### DISCUSSION TITLE
Multi 4in1 Kicad PCB version.

### DISCUSSION THREAD

**Discussion by sloped-soarer** [pascallanger/DIY-Multiprotocol-TX-Module#890] [General]
Hi.
Some time ago i was playing around with the import non KiCad project function in KiCad V6.
I imported the Eagle CAD project for the V1.1 STM32 version.
Having  just picked up where i left off, i decided to do a bit of a re-spin.
I added a crystal instead of a resonator, micro USB, and shrunk some components (minimum size 0603 capacitors). 
The result i suppose is somewhat academic as it is impossible to buy the 4in1 RF module..

Anyhow here's the results.
Oh ! and i brought out three unused pins to test points and added the slots for the RF module.

![front](https://github.com/pascallanger/DIY-Multiprotocol-TX-Module/assets/14750198/6ab05973-8ec0-4934-8605-e45e7177e54e)

![rear](https://github.com/pascallanger/DIY-Multiprotocol-TX-Module/assets/14750198/bcac230b-1415-4384-90b7-2e832552e9d3)

I don't know whether i'm going to send this to the PCB Fab house just yet.
Any comments welcome.

**Comment by sloped-soarer** [comment]
Me again ... Did anyone use SJ301 as it was intended, or is it more useful now to short out the diode D302 for use with transmitters without pullup resistors.

**Reply by pascallanger** [reply]
I don't think SJ301 has ever been used. It was there in case you don't have the XOR component in place. You can just get rid of it... I think the norm now is to send the signal inverted to the radio. So the schematic could be simplified using a simple transistor and a pull-up for the telemetry output. The telemetry input, while not use at all, could be done the same way as the PPM input.
By the way your PCB looks great. I have to learn kicad at some point.
